client/agent-opentracing/src/main/java/org/hawkular/apm/agent/opentracing/OpenTracingManager.java

```diff
@@ -460,6 +460,24 @@ public void setVariable(String name, Object value) {
     }
 
     /**
+     * This method determines whether the supplied object is an
+     * instance of the supplied class/interface.
+     *
+     * @param obj The object
+     * @param clz The class
+     * @return Whether the object is an instance of the class
+     */
+    public boolean isInstanceOf(Object obj, Class<?> clz) {
```

Comment:
Shouldn't this be called `isAssignableFrom`? There is also `Clazz.class.isInstance(obj)`